scripts/configs/drone/main.jsonnet

```diff
@@ -87,25 +87,38 @@ local docker_step(tag, os, arch, version='') =
     settings: {
       repo: docker_repo,
       dry_run: 'true',
+      target: if with_browser then 'with-browser' else 'release',
       build_args: [
         'TARGETPLATFORM=' + platform,
         'TARGETOS=' + os,
         'TARGETARCH=' + arch,
       ] + if std.length(version) > 0 then [
         'TARGETVARIANT=' + version,
+      ] else []
+       + if with_browser then [
+        'WITH_BROWSER=true',
       ] else [],
     },
   };
 
-local docker_build(os, arch, version='') =
-  docker_step('docker build', os, arch, version)
+local docker_build(os, arch, version='', with_browser=false) =
+  local tag = if with_browser then
+      'docker build (with browser)'
+        else
+      'docker build';
```

Comment:
Not sure I get what's going on here. Why are we setting `tag` to a string with spaces and parentheses? If this isn't the image tag I'd take this chance to add a comment here.

Reply:
Yes this isn't the image tag, I was following the existing use of the `tag` param in this jsonnet file. I will rename this in the new usages to avoid confusion